site/argument-file.md

```diff
@@ -23,4 +23,4 @@ imagetool @/path/to/build_args
 ```
 
 ### Copyright
-Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2019, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I thought the new copyright rule was: There is no need to update existing notices just to make this modification, but you should not use “All rights reserved” in new notices that you create or existing notices that you are updating for other reasons.

Reply:
right, I will remove the "all rights reserved".  The documentation was updated in previous PRs but the copyright wasn't updated.  I reviewed the files prior to cutting the next release and thought they needed to be updated.